features/product/managing_product_attributes/adding_non_translatable_attribute.feature

```diff
@@ -0,0 +1,18 @@
+@managing_product_attributes
+Feature: Adding a new non translatable product attribute
+    In order to add non translatable attribute
+    As an Administrator
+    I want to be able to toggle translatable checkbox
+
+    Background:
+        Given the store is available in "English (United States)"
+        And I am logged in as an administrator
+
+    @ui
+    Scenario: Adding a new non translatable product attribute
+        Given I want to create a new integer product attribute
+        When I specify its code as "damage"
```

Comment:
`I want to...` should be a `When` step instead of `Given`. Would be nice to fix that here and in other added scenarios in this PR
```suggestion
        When I want to create a new integer product attribute
        And I specify its code as "damage"
```